docs/pages/development/installation.md

```diff
@@ -156,7 +123,25 @@ import App from "./App";
 
 ### Loading published updates
 
-Development builds can also open and preview published updates to your app. To add this feature, add `expo-updates@0.8.0` or newer to your app if it isn't already installed, and add a small additional integration in your **AppDelegate.m** and **MainApplication.java** files.
+Development builds can also open and preview published updates to your app.
+
+<Tabs tabs={["SDK 45+", "SDK below 45"]}>
+
+<Tab>
+
+To add this feature, install `expo-updates` in your project:
+
+<Terminal cmd={["$ npx expo install expo-updates"]} />
+
+You may need to rerun the config plugin for `expo-dev-client` after doing so.
```

Comment:
```suggestion
After doing so, you may need to re-run the config plugin for `expo-dev-client`.
```

I don't get the context of why this step is a "may"? Is it like some users have to and some don't if they install `expo-updates`?

Reply:
Thanks for making me think about this a little harder -- I realized that this step isn't actually necessary at all! 😅  Rewrote a few other places to reflect this, as well.

Reply:
Thanks for making these changes 🙏 
Love the new change "SDK 44 and below". I think it is more clear to imply the info docs are conveying there.